src/main/java/uk/gov/ch/developer/docs/controller/developer/SignOutController.java

```diff
@@ -1,57 +1,44 @@
 package uk.gov.ch.developer.docs.controller.developer;
 
 import java.io.IOException;
-import java.util.Map;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
 import uk.gov.ch.developer.docs.DocsWebApplication;
+import uk.gov.ch.oauth.Oauth2;
 import uk.gov.ch.oauth.identity.IIdentityProvider;
 import uk.gov.companieshouse.logging.Logger;
 import uk.gov.companieshouse.logging.LoggerFactory;
 import uk.gov.companieshouse.session.Session;
-import uk.gov.companieshouse.session.SessionKeys;
 import uk.gov.companieshouse.session.handler.SessionHandler;
 import uk.gov.companieshouse.session.store.Store;
 
 @Controller
 public class SignOutController {
     private static final Logger LOGGER =
             LoggerFactory.getLogger(DocsWebApplication.APPLICATION_NAME_SPACE);
-    private static final String SIGN_IN_INFO = SessionKeys.SIGN_IN_INFO.getKey();
 
     @Autowired
     private IIdentityProvider identityProviders;
     @Autowired
     private Store store;
+    @Autowired
+    private Oauth2 oauth2;
+
+    public SignOutController() {
```

Comment:
Why have we got this empty constructor?

Reply:
I removed the constructor and then ran the tests and they still passed so I committed the change.